services/facility_queue/facility_queue.py

```diff
@@ -220,7 +221,12 @@ def service(queue):
                             ),
                         )
 
-                        if response.status_code == 200:
+                        if "Zero records returned" in str(response.text):
```

Comment:
Should status also be updated here?

Reply:
No, this happens if your FP account has nothing in it yet. Which always happens if you dont send any requests for a bit to IPAC, and then start sending stuff. It's something to ignore, just their webpage showing a string instead of an empty list...

Reply:
Makes sense.